packages/ui/sass/components/buttons.scss

```diff
@@ -89,6 +89,10 @@ $white-tint: rgba(255, 255, 255, 0.4);
     cursor: default;
   }
 
+  &--disabled-color {
```

Comment:
I suggest changing this modifier's name to `loading`.

Also, in this case, we should not change the button's border, instead we should add `pointer-events: none`.